functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java

```diff
@@ -70,6 +70,7 @@ private BuiltProject executeGoals(String... goals) {
                     .setProperties(asProperties(buildConfigurator.getSystemProperties()))
                     .skipTests(buildConfigurator.isSkipTestsEnabled())
                     .setMavenOpts(buildConfigurator.getMavenOpts())
+                    .setAlsoMake(true)
```

Comment:
This has been already committed to the master branch: 
https://github.com/arquillian/smart-testing/commit/cf6f1d0c153ce1bfc7c3c91218f2298e487bb7ce

Reply:
I know, I cherry-picked from here. Makes me wonder why it still shows up though...